src/Helper/OrganisationApiHelper.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Drupal\os2forms_organisation\Helper;
+
+use Symfony\Component\HttpClient\CurlHttpClient;
+
+/**
+ * Organisation API helper class.
+ */
+class OrganisationApiHelper {
+
+  /**
+   * Curl Client.
+   *
+   * @var \Symfony\Component\HttpClient\CurlHttpClient
+   */
+  private ?CurlHttpClient $client = NULL;
+
+  /**
+   * The Settings.
+   *
+   * @var \Drupal\os2forms_organisation\Helper\Settings
+   */
+  private Settings $settings;
+
+  /**
+   * Constructor.
+   */
+  public function __construct(Settings $settings) {
+    $this->settings = $settings;
+  }
+
+  /**
+   * Get curl client.
+   */
+  private function getCurlClient(): CurlHttpClient {
+    if ($this->client) {
+      return $this->client;
+    }
+    else {
+      $this->client = new CurlHttpClient();
+
+      return $this->client;
+    }
+  }
+
+  /**
+   * Get bruger informationer for bruger.
+   *
+   * @phpstan-return array<string, mixed>
+   */
+  public function getBrugerInformationer(string $brugerId): array {
+    $client = $this->getCurlClient();
+
+    $this->settings->getOrganisationApiEndpoint();
+
+    $response = $client->request('GET', $this->settings->getOrganisationApiEndpoint() . 'bruger/' . $brugerId);
+
+    if (200 === $response->getStatusCode()) {
+      return $response->toArray();
+    }
+    else {
+      return [];
+    }
```

Comment:
You don't need the `else` branch:

```suggestion
    }

    return [];
```

Reply:
Refactored code.